test/module/irohad/torii/torii_transport_command_test.cpp

```diff
@@ -219,10 +220,10 @@ TEST_F(CommandServiceTransportGrpcTest, ListToriiPartialInvalid) {
 
   size_t counter = 0;
   EXPECT_CALL(*proto_tx_validator, validate(_))
-      .Times(kTimes)
+      .Times(AtLeast(1))
       .WillRepeatedly(Return(shared_model::validation::Answer{}));
   EXPECT_CALL(*tx_validator, validate(_))
-      .Times(kTimes)
+      .Times(AtLeast(1))
```

Comment:
sab. the counter in handler should reach `kTimes`, so the number of calls expectation should also be `kTimes`